### PR TITLE
Merge childrenIncome into children in React store

### PR DESCRIPTION
**Pull request by awong-dev**
This makes it much easier to manipulate the model since there are not
any parellel arrays.

Parallel arrays === bad.

### CODE REVIEW

File: src/client/components/household-information/AnnualIncomeSection.jsx [hunk 1]
```diff
@@ -17,49 +17,42 @@ function getErrorMessage(field, message) {
...
-          <div key={reactKey++}>
-            <h6>Child: {`${childrenData[index].childFullName.first.value} ${childrenData[index].childFullName.last.value}`}</h6>
+          <div key={`child-${index}`}>
+            <h6>Child: {`${child.childFullName.first.value} ${child.childFullName.last.value}`}</h6>
             <table className="review usa-table-borderless">
               <tbody>
                 <tr>
                   <td>Children Gross Income:</td>
-                  <td>{obj.childGrossIncome.value}</td>
+                  <td>{child.income.grossIncome.value}</td>
```
Comment (akainic) [inline src/client/components/household-information/AnnualIncomeSection.jsx]: I think these are incorrect, looks like it's just `child.grossIncome` based on the store?

**Comment by akainic** [comment]
LGTM with one small fix

**Comment by robbiethegeek** [comment]
I agree with @akainic if  you change and then push to this branch I will LGTM.

**Comment by robbiethegeek** [comment]
LGTM

**Pull request merged**